modules/tinymce/src/core/test/ts/webdriver/paste/CopyAndPasteTest.ts

```diff
@@ -0,0 +1,177 @@
+import { Cursors, RealClipboard, RealMouse } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyAssertions, TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('webdriver.tinymce.core.paste.CopyAndPasteTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    toolbar: false,
+    statusbar: false
+  }, []);
+
+  const pCopyAndPaste = async (editor: Editor, source: Cursors.CursorPath, target: Cursors.CursorPath): Promise<void> => {
+    TinySelections.setSelection(editor, source.startPath, source.soffset, source.finishPath, source.foffset);
+    // at the moment: RealClipboard.pCopy('iframe => body'), doesn't work in with all browser (see https://github.com/webdriverio/webdriverio/issues/622)
+    // chrome, chrome-headless, firefox-headless -> it doesn't work
+    // firefox -> it works
+    TinyUiActions.clickOnMenu(editor, 'button:contains("Edit")');
+    await TinyUiActions.pWaitForUi(editor, '*[role="menu"]');
+    await RealMouse.pClickOn('div[title="Copy"]');
+    TinySelections.setSelection(editor, target.startPath, target.soffset, target.finishPath, target.foffset);
+    if (PlatformDetection.detect().browser.isSafari()) {
```

Comment:
I wonder if we should just do this logic for all cases, as it doesn't really matter which means is used 🤷

Reply:
Actually, it does not work in all cases some browsers forbid using it by saying that direct access to the clipboard is not supported

Reply:
Ahh right forgot that sorry. Lets leave it forking then 👍